go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/kubernetes-csi/csi-proxy/client v1.0.1
```

Comment:
1.0.2?

Reply:
should be fixed after https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/276/ is merged